subprojects/docs/src/docs/release/notes.md

```diff
@@ -77,31 +77,6 @@ ADD RELEASE FEATURES ABOVE
 
 -->
 
-#### Improved Gradle User Home Cache Cleanup
-Previously, cleanup of the caches in Gradle User Home used fixed retention periods (30 days or 7 days depending on the cache).  
-These retention periods can now be configured via the [Settings](dsl/org.gradle.api.initialization.Settings.html) object in an init script in Gradle User Home.
-
-```groovy
-beforeSettings { settings ->
-    settings.caches {
-        downloadedResources.removeUnusedEntriesAfterDays = 45
-    }
-}
-```
-
-Furthermore, it was previously only possible to partially disable cache cleanup via the `org.gradle.cache.cleanup` Gradle property in Gradle User Home.  
-Disabling cache cleanup now affects more caches under Gradle User Home and can also be configured via the [Settings](dsl/org.gradle.api.initialization.Settings.html) object in an init script in Gradle User Home.
-
-```groovy
-beforeSettings { settings ->
-    settings.caches {
-        cleanup = Cleanup.DISABLED
-    }
-}
-```
-
-See [Configuring cleanup of caches and distributions](userguide/directory_layout.html#dir:gradle_user_home:configure_cache_cleanup) for more information.
-
```

Comment:
This was left over in the release notes from 8.0 and didn't get removed when the 8.1 notes were cut.  Just happened to notice it while resolving the conflict.

Reply:
good catch